src/style.css

```diff
@@ -1,3 +1,68 @@
+html {
+  --active-nav-color: #00acdc;
+}
+
 body {
+  font:
+    500 12px / 14px Roboto,
+    sans-serif;
+}
+
+ul {
   margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img-wrapper {
+  width: 40px;
+  height: 40px;
+}
+
+.img {
+  width: 100%;
+}
+
+.nav-list {
+  display: flex;
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.nav-link:hover {
```

Comment:
Both this and the next selector has the same style. So you can combine this style and the next one into a single expression, listing selectors as comma-separated list.